sql/src/test/java/io/crate/integrationtests/PostgresStatsTablesITest.java

```diff
@@ -0,0 +1,178 @@
+            assertThat(results, is(new int[] {1, 1}));
+
+            ResultSet resultSet = conn.prepareStatement("select stmt from sys.jobs_log " +
+                                                        "where stmt='" + stmtStr1Where + "' OR " +
+                                                        "stmt='" + stmtStr2Where + "' " +
+                                                        "order by started").executeQuery();
+            assertThat(resultSet.next(), is(true));
+            assertThat(resultSet.getString(1), is(stmtStr1));
+            assertThat(resultSet.next(), is(true));
+            assertThat(resultSet.getString(1), is(stmtStr2));
+        }
+    }
+
+    @Test
+    public void testStatsTableFailure() throws Exception {
+        try (Connection conn = DriverManager.getConnection(JDBC_POSTGRESQL_URL)) {
+            conn.setAutoCommit(true);
+            conn.prepareStatement("create table t (a integer not null, b string)").execute();
+
+            String uniqueId = UUID.randomUUID().toString();
+            String stmtStr = "insert into t(a,b) values(null, '" + uniqueId + "')";
+            String stmtStrWhere = "insert into t(a,b) values(null, ''" + uniqueId + "'')";
+            try {
+                conn.prepareStatement(stmtStr).execute();
+                fail("NOT NULL constraint is not respected");
+            } catch (Exception e) {
+                ResultSet resultSet = conn.prepareStatement("select stmt, error from sys.jobs_log " +
+                                                            "where stmt='" + stmtStrWhere + "'").executeQuery();
+                assertThat(resultSet.next(), is(true));
+                assertThat(resultSet.getString(1), is(stmtStr));
+                assertThat(resultSet.getString(2), is("Cannot insert null value for column a"));
+            }
+        }
+    }
+
+    @Test
+    public void testBatchOperationStatsTableFailure() throws Exception {
+        try (Connection conn = DriverManager.getConnection(JDBC_POSTGRESQL_URL)) {
+            conn.setAutoCommit(true);
+            conn.createStatement().executeUpdate("create table t (a integer not null, x string) " +
+                                                 "with (number_of_replicas = 0)");
+            ensureYellow();
+
+            String uniqueId1 = UUID.randomUUID().toString();
+            String uniqueId2 = UUID.randomUUID().toString();
+            String uniqueId3 = UUID.randomUUID().toString();
+            Statement statement = conn.createStatement();
+            String stmtStr1 = "insert into t (a, x) values (1, '" + uniqueId1 + "')";
+            String stmtStr1Where = "insert into t (a, x) values (1, ''" + uniqueId1 + "'')";
+            String stmtStr2 = "insert into t (a, x) values (null, '" + uniqueId2 + "')";
+            String stmtStr2Where = "insert into t (a, x) values (null, ''" + uniqueId2 + "'')";
+            String stmtStr3 = "insert into t (a, x) values (3, '" + uniqueId3 + "')";
```

Comment:
I want to have the uniqueIds in the sys.jobs_log table to avoid possible interference with other tests. If I use parameter substitution then the entries in the jobs_log will be like:
insert into t (a, x) values ($1, $2)